tests/clob.ts

```diff
@@ -11,11 +11,15 @@ describe("clob", () => {
   const provider = anchor.AnchorProvider.env();
   anchor.setProvider(provider);
 
-  const payer = provider.wallet.payer;
+  const payer = anchor.web3.Keypair.generate();
```

Comment:
why introduce a separate payer?

the benefit of the in-built Anchor one is that it will automatically sign transactions, so you don't have to do the `.signers([payer])` biz

Reply:
The problem is that `payer` apparently does not exist in `provider.wallet`. I can however change it to default to Anchor's signer implicitly

Reply:
yeah, that would be good IMO

the way I deal with this in the scripts is `const payer = provider.wallet['payer'];`

Reply:
Looking into it, there is a `payer` only if it's a `NodeWallet`, which `BankrunProvider` provides but not necessarily `anchor.AnchorProvider.env()`. I think that would require switching to Bankrun for `tests/clob.ts` to make it correctly, but that would be tedious in this PR. I can just revert the payer generation for now, and address it in another PR later

Reply:
ok, I'll merge it as is, and if it starts feeling non-ergonomic we can always roll back later